.github/workflows/build_and_publish.yml

```diff
@@ -48,22 +48,35 @@ jobs:
     - name: Check out git repository
       uses: actions/checkout@v4
 
-    - name: Publish main image (Dockerfile) to Registry
-      uses: elgohr/Publish-Docker-Github-Action@v5
+    - name: Login to Docker Hub
+      uses: docker/login-action@v3
       with:
-        name: clinicalgenomics/scout
         username: ${{ secrets.DOCKER_USERNAME }}
         password: ${{ secrets.DOCKER_PASSWORD }}
-        tags: "latest,${{ github.event.release.tag_name }}"
+
+    - name: Set up Docker Buildx
+      id: buildx
+      uses: docker/setup-buildx-action@v3
 
     - name: Publish server image (Dockerfile-server) to Registry
-      uses: elgohr/Publish-Docker-Github-Action@v5
+      uses: docker/build-push-action@v6
       with:
-        name: clinicalgenomics/scout-server
-        username: ${{ secrets.DOCKER_USERNAME }}
-        password: ${{ secrets.DOCKER_PASSWORD }}
-        dockerfile: Dockerfile-server
-        tags: "latest,${{ github.event.release.tag_name }}"
+        context: ./
+        file: ./Dockerfile-server
+        push: true
+        provenance: mode=max
```

Comment:
I want to use this action instead because provenance and sbom are well documented: https://docs.docker.com/build/ci/github-actions/attestations/